lib/hammer_cli_foreman/host.rb

```diff
@@ -213,15 +213,14 @@ def send_request
       end
     end
 
+    class ConfigReportsCommand < HammerCLIForeman::ListCommand
+      command_name 'config-reports'
```

Comment:
Hmm, we should probably keep the `reports` command with a deprecation warning for the sake of compatibility.